src/basic/missing_fs.h

```diff
@@ -116,3 +116,58 @@ assert_cc(FS_KEY_DESCRIPTOR_SIZE == 8);
 #ifndef FILEID_KERNFS
 #define FILEID_KERNFS 0xfe
 #endif
+
+/* linux/fsverity.h */
```

Comment:
Please copy the header into basic/linux, rather than adding entries in missing_fs.h .

Reply:
Great.  I was just about to push again.  Thanks for the advice!